isort/settings.py

```diff
@@ -48,6 +49,10 @@
 MAX_CONFIG_SEARCH_DEPTH = 25  # The number of parent directories isort will look for a config file within
 DEFAULT_SECTIONS = ('FUTURE', 'STDLIB', 'THIRDPARTY', 'FIRSTPARTY', 'LOCALFOLDER')
 
+safety_exclude_re = re.compile(
+    r"/(\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|_build|buck-out|build|dist|lib/python\d.\d)/"
```

Comment:
I guess this assumes there will not be a Python 3.10. Is that documented somewhere as the case?

`\d` includes all Unicode digits so I find `[0-9]` a safer alternative.

https://docs.python.org/3/library/re.html#index-27

Reply:
Oh, hey, yeah, good point. I didn't think about 3.10+ at all.